<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlobCountWidget</class>
 <widget class="QWidget" name="BlobCountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>681</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>单个连通域面积</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>41</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上限</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下限</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSingleBlobAreaMax">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>39</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="cursorPosition">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSingleBlobAreaMin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>180</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Blob数量</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>41</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上限</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>下限</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBlobCountMax">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>39</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBlobCountMin">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCalcValue">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>21</width>
     <height>480</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>640</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>计算门限</string>
   </property>
   <widget class="QPushButton" name="pushButtonCalc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>35</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAuto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>自动设置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
